.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>microclimate-eclipse-tools</name>
```

Comment:
Should it be codewind-eclipse? I assume other places using 'microclimate' will be refactored later.

Reply:
Yes.  Thanks for catching that.  Fixed now.